calicoctl.py

```diff
@@ -291,59 +302,23 @@ def node(ip, force_unix_socket, node_image, ip6=""):
     client.create_host(hostname, ip, ip6)
 
     # Enable IP forwarding since all compute hosts are vRouters.
+    # IPv4 forwarding should be enabled already by docker.
     sysctl("-w", "net.ipv4.ip_forward=1")
     sysctl("-w", "net.ipv6.conf.all.forwarding=1")
 
-    # The docker daemon could be in one of two states:
-    # 1) Listening on /var/run/docker.sock - the default
-    # 2) listening on /var/run/docker.real.sock - if it's been previously run
-    #    with --force-unix-socket
-    enable_socket = "NO"
-
     # We might need to talk to a different docker endpoint, so create some
     # client flexibility.
     node_docker_client = docker_client
 
-    if force_unix_socket:
-        # Update docker to use a different unix socket, so powerstrip can run
-        # its proxy on the "normal" one. This provides simple access for
-        # existing tools to the powerstrip proxy.
-
-        # Set the docker daemon to listen on the docker.real.sock by updating
-        # the config, clearing old sockets and restarting.
-        socket_config_exists = \
-            DOCKER_OPTIONS in open(DOCKER_DEFAULT_FILENAME).read()
-        if not socket_config_exists:
-            with open(DOCKER_DEFAULT_FILENAME, "a") as docker_config:
-                docker_config.write(DOCKER_OPTIONS)
-            clean_restart_docker(REAL_SOCK)
-
-        # Always remove the socket that powerstrip will use, as it gets upset
-        # otherwise.
-        if os.path.exists(POWERSTRIP_SOCK):
-            os.remove(POWERSTRIP_SOCK)
-
-        # At this point, docker is listening on a new port but powerstrip isn't
-        # running, so docker clients need to talk directly to docker.
+    if DOCKER_OPTIONS in open(DOCKER_DEFAULT_FILENAME).read():
```

Comment:
need to surround this in a `try:` block or it will fail on systems where `DOCKER_DEFAULT_FILENAME` doesn't exist.